### PR TITLE
feat: Change player ships colors

### LINKED ISSUE #48: Modify-Ships-Color
## Problem Description

Modify the color of the ships.

## Todo List
-   [x] Set player1 ship color to red for better visibility.
-   [x] Set player2 ship color to blue for easy differentiation between players.
-   [x] Improve overall game visual clarity with distinct ship colors.

## Feature Description

Change the color of ship1(player1) to red and ship2(player2) to blue to distinguish each of them.

### PR DESCRIPTION
**Pull request by aeioiie**
## What
This PR adds the changes for users' better visibility. It includes the following changes:
-  Add a color constructor to Ship method in Ship.java.
-  Add a color red to Screen.java when initializing player 1.
-  Add a color blue to TwoPlayerMode.java when initializing player 2.

## Why
To clearly distinguish between player1 and player2 in two-player mode.

## Related Issue
Closes #48 

## How to Test
1. Start the application locally.
2. Select 2-player-mode.
3. Check if player1's ship color is red and player2's ship color is blue.

## Screenshots
image

## Additional Information
The color of ship1 and ship2 is the same with each of their life graphic color.

### CODE REVIEW

**Review by DongWonLee2 — APPROVED** [review]
image

LGTM

**Review by LeeHyunWoo02 — APPROVED** [review]
LGTM

File: src/entity/Ship.java [hunk 1]
```diff
@@ -50,8 +50,8 @@ public class Ship extends Entity {
 	 *            Initial position of the ship in the Y axis.
 	 */
 	//Edit by Enemy
-	public Ship(final int positionX, final int positionY) {
-		super(positionX, positionY - 50, 13 * 2, 8 * 2, Color.GREEN);
+	public Ship(final int positionX, final int positionY, final Color color) {
```
Comment (yunsful) [inline src/entity/Ship.java:53]: I think it’s a good idea to add a color variable to the Ship class constructor, allowing you to customize the color of the ship as desired.

**Review by yunsful — APPROVED** [review]
image

LGTM

**Review by DongWonLee2 — APPROVED** [review]

File: src/entity/Ship.java (hunk 1 above)
Comment (aeioiie) [reply src/entity/Ship.java:53]: Thank you.

**Pull request merged**